tests/installation/bootloader_s390.pm

```diff
@@ -95,6 +95,14 @@ sub prepare_parmfile {
                   shorten_url($host . '/' . get_required_var('REPO_0') . "/LiveOS/squashfs.img") :
                   $host . '/' . get_var('REPO_999'));
             $params .= $root_line;
+
+            # add mandatory boot params
+            $params .= ' cio_ignore=all,!condev,!0.0.0150';
+            $params .= ' hvc_iucv=8';
+            $params .= " live.password=$testapi::password";
+
+            # add optional boot params
+            $params .= ' rd.zdev=dasd,0.0.0150' if (get_var('AGAMA_ACTIVE_DASD'));
```

Comment:
this is wrong, AGAMA_ACTIVE_DASD means that Agama activates the DASD ... it would be `unless` (by default DASD is activated via paramfile).
could you rename it to `AGAMA_ACTIVATE_DASD` please.

Reply:
Changed to ACTIVATE